docs/src/index.ejs

```diff
@@ -26,27 +26,24 @@
   </script>
 </head>
 <body>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.1/anchor.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.1/anchor.min.js"></script>
   <script
-    crossOrigin="true"
```

Comment:
The `crossorigin` attribute let's us see errors that are thrown in these scripts.  Without it, the error will not be logged in the console and debugging is impossible.

When copied from Semantic UI React's `Document.js` I failed to change the camel cased `crossOrigin` to `crossorigin`.  If this is fixed, is Edge happy?  We'd like to have these properties still.